bom/pom.xml

```
@@ -122,6 +122,16 @@
                 <artifactId>quarkus-amazon-ssm-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkiverse.amazonservices</groupId>
+                <artifactId>quarkus-amazon-cognito-idp</artifactId>
```

```suggestion
                <artifactId>quarkus-amazon-cognito-idp</artifactId>
```
To be more consistent with the rest, I would just call the extension `quarkus-amazon-cognito`. AFAICS, Cognito is just that right? Or you expect it to require some additional extensions later?

---

Cognito have three API (https://docs.aws.amazon.com/cognito/index.html). These APIs do not have stable names (Cognito Sync has no such problem).

| Docs name | SDK package  | aws-cli subcommand |
|-----------------|---------------------|-----------------------------|
| [Amazon Cognito user pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html) | [cognitoidentityprovider](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/cognitoidentityprovider/package-frame.html) | [`cognito-idp`](https://docs.aws.amazon.com/cli/latest/reference/cognito-idp/index.html) |
| [Amazon Cognito identity pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-identity.html) | [cognitoidentity](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/cognitoidentity/package-frame.html) | [`cognito-identity`](https://docs.aws.amazon.com/cli/latest/reference/cognito-identity/index.html) |

I chose the name `cognito-idp` as the shortest and most readable.

---

@c0va23 The naming situation is unfortunate. I must say that after looking at the linked website, when I see "cognito-idp" I think it's for "identity pools".

My suggestion is to use `cognito-user-pools`, `cognito-identity-pools` and `cognito-sync`.

---

@kdubb, i rename extension to `cognito-user-pools`.